### ISSUE TITLE
Wallet will not load and cannot restore wallet.

### ISSUE THREAD

**Issue by Loppops** [Multibit-Legacy/multibit-hd#1011]
This problem began yesterday. I'd recently received some Bitcoin and tried to open my wallet. My password was accepted, it synchronised, but would not let me click the 'Finish' button, either during synchronisation or after. The attached picture shows this problem. I closed it down and reattempted it several times with no luck. Restarting my computer did not help either. 

![multibiterror](https://user-images.githubusercontent.com/29780077/27697193-7b6aeefe-5ceb-11e7-942a-ccba23f544cf.png)

I then tried to restore my wallet, using the most recent back up. It then just displayed a cross next to 'Wallet created' (relevant picture attached). It seems to synchronise 'successfully' (although it takes a very, very long time), but then I'm back to square one with a wallet that won't open. I'm wary of using older backups in case I lose my bitcoin, since it was a recent transaction.

![multibitrestorewallet](https://user-images.githubusercontent.com/29780077/27697233-8f8b3c68-5ceb-11e7-81c1-074905674276.png)

So I don't have a clue what's going on or what to do. I should say I'm very much a novice when it comes to all this, although I've used Multibit for over a year now, so I apologise in advance if I have trouble understanding any solutions there might be. I've tried everything I can so far.

Thanks in advance, and I'm sorry to bother you all with this.

**Comment by linhaifan** [comment]
I meet the same problem. What's the solution of it?

**Comment by DMFZ** [comment]
Sort of the same issue here. In my case it errors out at the password
![errpass](https://user-images.githubusercontent.com/15316961/27982854-17461862-6363-11e7-9778-77f3adbb421a.jpg)
They said it's a known bug that is causing the error and they're working on it. You can use Breadwallet or Simple Bitcoin wallet, and restore with your seed words; only if you have 12 words, if you have > than that, then you just have to wait. Hope this helps.

**Comment by Loppops** [comment]
Yeah, I got the same reply from them as well. I've never used either of those two programs before, and I don't really use Android, so it's not ideal, but I'll have to give it a shot. It must be better than waiting, at least...

**Comment by frederickmorty** [comment]
If you have wallet ,aes + right password thats all thats needed to get your money.
seed words are just an extra precaution if you loose files and password!

why is this still not addressed? my funds are also stuck.

**Comment by DMFZ** [comment]
hey guys. have you received any update from these people!? it's been a while... i wonder WTF is going on..

**Comment by Loppops** [comment]
They said they'd be in touch when they've fixed it, but I've heard nothing yet. It's been over three weeks since the problem started for me. I don't know what they're playing at...should've been fixed by now.

**Comment by andyg51** [comment]
I have had the same type of problem, password unlocks, but the wallet stays in loading mode and doesn't
fully load up. Some sort of help on the matter would be appreciated.

**Comment by protemson1** [comment]
@andyg51 
@Loppops 
@linhaifan 
 I can recover multibit bitcoin wallet , before, I have recovered several multibit bitcoin wallets for my customers, you can find me at : walletsrecoveryservice@gmail.com

**Comment by doge2021** [comment]
> I have had the same type of problem, password unlocks, but the wallet stays in loading mode and doesn't fully load up. Some sort of help on the matter would be appreciated.
if you have the seed, can try to restore via electrum wallet. if not, there is bug caused by this bad wallet.
[multibit wallet problems has details
](https://btc2doge.com/tag/multibit-hd-old-wallet-restore/)

**Comment by doge2021** [comment]
> I meet the same problem. What's the solution of it?

if you have the seed, can try to restore via electrum wallet. if not, there is bug caused by this bad wallet.
[multibit wallet problems has details
](https://btc2doge.com/tag/multibit-hd-old-wallet-restore/)

**Comment by doge2021** [comment]
> This problem began yesterday. I'd recently received some Bitcoin and tried to open my wallet. My password was accepted, it synchronised, but would not let me click the 'Finish' button, either during synchronisation or after. The attached picture shows this problem. I closed it down and reattempted it several times with no luck. Restarting my computer did not help either.
> 
> ![multibiterror](https://user-images.githubusercontent.com/29780077/27697193-7b6aeefe-5ceb-11e7-942a-ccba23f544cf.png)
> 
> I then tried to restore my wallet, using the most recent back up. It then just displayed a cross next to 'Wallet created' (relevant picture attached). It seems to synchronise 'successfully' (although it takes a very, very long time), but then I'm back to square one with a wallet that won't open. I'm wary of using older backups in case I lose my bitcoin, since it was a recent transaction.
> 
> ![multibitrestorewallet](https://user-images.githubusercontent.com/29780077/27697233-8f8b3c68-5ceb-11e7-81c1-074905674276.png)
> 
> So I don't have a clue what's going on or what to do. I should say I'm very much a novice when it comes to all this, although I've used Multibit for over a year now, so I apologise in advance if I have trouble understanding any solutions there might be. I've tried everything I can so far.
> 
> Thanks in advance, and I'm sorry to bother you all with this.
if you have the seed, can try to restore via electrum wallet. if not, there is bug caused by this bad wallet.
[multibit wallet problems has details
](https://btc2doge.com/tag/multibit-hd-old-wallet-restore/)
